src/content-script.ts

```diff
@@ -41,7 +41,6 @@ import { isBodyWithReplies, isRepositoryDetails } from "./validations";
 	// 5. Parse the replies body as yml
 	const repliesConfiguration = yaml.load(repliesBody);
 	if (!isBodyWithReplies(repliesConfiguration)) {
-		console.error("Invalid saved replies:", repliesConfiguration);
```

Comment:
[Bug] Hmm, what if the response is like a 400 or 500? I think we'll still want to `console.error` in that case.

Reply:
Ahh yes, thank you for pointing that out! PR updated!

Reply:
@JoshuaKGoldberg Do we want to log failed fetch for the other fetch calls? If so, I can open an issue for those as well.